PractiseProject/Bai23/Program.cs

```diff
@@ -0,0 +1,12 @@
+﻿
+Console.WriteLine("nhap so nguyen duong n :");
+int n = Convert.ToInt32(Console.ReadLine());
+for(int i = 1; i < n; i++) // vong lap for , chay tu 1 den so n 
+{
+    if(i%3==0)
+    {
+        Console.WriteLine("Cac so chia het cho 3 :"+i);
```

Comment:
This line not correct. You should store these number in array and show it in one line. Not duplicate ""Cac so chia het cho 3 :". In this sentence, "Các" mean many number, not one